test/powershell/Language/Parser/Conversions.Tests.ps1

```diff
@@ -31,27 +31,30 @@ Describe 'conversion syntax' -Tags "CI" {
                 ## would become `, [List[int]]@(1,2)`, which is more confusing than `$result = [List[int]]@(1,2)`.
                 ## This is why the current form of `$result = [List[int]]@(1,2)` is used intentionally here.
 
```

Comment:
VSCode formats this.
Is it OK to keep this formatting?
Or is there a setting that aligns one line hashtables?